doc/classes/Node.xml

```diff
@@ -154,32 +154,34 @@
 			<param index="0" name="sibling" type="Node" />
 			<param index="1" name="force_readable_name" type="bool" default="false" />
 			<description>
-				Adds a [param sibling] node to current's node parent, at the same level as that node, right below it.
+				Adds a [param sibling] node to this node's parent, and moves the added sibling right below this node.
 				If [param force_readable_name] is [code]true[/code], improves the readability of the added [param sibling]. If not named, the [param sibling] is renamed to its type, and if it shares [member name] with a sibling, a number is suffixed more appropriately. This operation is very slow. As such, it is recommended leaving this to [code]false[/code], which assigns a dummy name featuring [code]@[/code] in both situations.
 				Use [method add_child] instead of this method if you don't need the child node to be added below a specific node in the list of children.
-				[b]Note:[/b] If this node is internal, the new sibling will be internal too (see [code]internal[/code] parameter in [method add_child]).
+				[b]Note:[/b] If this node is internal, the added sibling will be internal too (see [method add_child]'s [code]internal[/code] parameter).
 			</description>
 		</method>
 		<method name="add_to_group">
 			<return type="void" />
 			<param index="0" name="group" type="StringName" />
 			<param index="1" name="persistent" type="bool" default="false" />
 			<description>
-				Adds the node to a group. Groups are helpers to name and organize a subset of nodes, for example "enemies" or "collectables". A node can be in any number of groups. Nodes can be assigned a group at any time, but will not be added until they are inside the scene tree (see [method is_inside_tree]). See notes in the description, and the group methods in [SceneTree].
-				The [param persistent] option is used when packing node to [PackedScene] and saving to file. Non-persistent groups aren't stored.
-				[b]Note:[/b] For performance reasons, the order of node groups is [i]not[/i] guaranteed. The order of node groups should not be relied upon as it can vary across project runs.
+				Adds the node to the [param group]. Groups can be helpful to organize a subset of nodes, for example [code]"enemies"[/code] or [code]"collectables"[/code]. See notes in the description, and the group methods in [SceneTree].
+				If [param persistent] is [code]true[/code], the group will be stored when saved inside a [PackedScene]. All groups created in the Node dock are persistent.
+				[b]Note:[/b] To improve performance, the order of group names is [i]not[/i] guaranteed and may vary between project runs. Therefore, do not rely on the group order.
+				[b]Note:[/b] [SceneTree]'s group methods will [i]not[/i] work on this node if not inside the tree (see [method is_inside_tree]).
```

Comment:
The fact that groups added via `add_to_group()` in a `@tool` script aren't displayed in the editor should be mentioned here. (This is intended to be used by editor plugins and stuff like that.)

Reply:
To clarify, the editor displays only persistent groups. `@tool` or not does not matter.

Reply:
I believe I solved this with a simple "_and displayed_" in this last push